Sources/Core/Common/SDKSettings.swift

```diff
@@ -148,7 +148,7 @@ public enum SDKSettings {
       }
 
       #if swift(>=4.1)
-      let behaviors: [SDKLoggingBehavior] = FBSDKSettings.loggingBehavior().compactMap(createBehavior)
+      let behaviors: [SDKLoggingBehavior] = FBSDKSettings.loggingBehaviors.compactMap(createBehavior)
```

Comment:
What is the purpose of this change? I believe that loggingBehavior remains a function in `v4.42.0`.

Reply:
In Swift5 FBSDKSettings.loggingBehavior().compactMap(createBehavior) as been renamed FBSDKSettings.loggingBehaviors.compactMap(createBehavior)